### PR TITLE
add support for solana sdk keypair

### PR DESCRIPTION
**Pull request by kstep**
Add signature verification facilities to solana_sdk types Keypair and Pubkey

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -31,8 +31,8 @@ sentry-tracing = { version = "0.27", optional = true }
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 serde_with = { version = "1.12", optional = true }
-solana-client = { version = "1.10", optional = true }
-solana-sdk = { version = "1.10", optional = true }
+solana-client = { version = "1.14", optional = true }
```
Comment (btolfa) [inline Cargo.toml:34]: I would suggest bump crate version - The changing of solana version can break dependent crates

**Review by btolfa — COMMENTED** [review]
I would suggest bump crate version - The changing of solana version can break dependent crates

File: src/crypto.rs [hunk 2]
```diff
@@ -39,6 +42,50 @@ impl PublicKeyExt for PublicKey {
     }
 }
 
+#[cfg(feature = "solana-sdk")]
```
Comment (BorovkovSergey) [inline src/crypto.rs:45]: Should this feature to be in cargo.toml features list?

Comment (kstep) [reply src/crypto.rs:45]: Actually, it's an "auto-feature" due to `solana-sdk` crate being optional. So here I just check if the dependency is on or off.

**Review by BorovkovSergey — APPROVED** [review]

**Pull request merged**